apps/charterafrica/src/components/OpportunityCards/OpportunityCards.js

```diff
@@ -0,0 +1,70 @@
+import { Typography, Box, Button } from "@mui/material";
+import React, { useState } from "react";
+
+import FeaturedPostCard from "../FeaturedPostCard/FeaturedPostCard";
+
+import OpportunityCardGrid from "./OpportunityCardGrid";
+import OpportunityCardHeader from "./OpportunityCardHeader";
+
+const OpportunityCards = React.forwardRef(function OpportunityCards(
+  props,
+  ref
+) {
+  const { config, items, title, sx } = props;
+
+  const [showAll, setShowAll] = useState(false);
+  const featuredEvent = items?.find((item) => item.featured);
+
+  if (!items?.length) {
+    return null;
+  }
+  return (
+    <Box
+      ref={ref}
+      sx={{
+        ...sx,
+      }}
+    >
+      <OpportunityCardHeader
+        title={title}
+        onClick={() => setShowAll(!showAll)}
+        showAll={showAll}
+        showAllText={config?.showAllText || "Show all"}
+        showLessText={config?.showLessText || "Show less"}
+      />
+      {featuredEvent ? <FeaturedPostCard {...featuredEvent} /> : null}
```

Comment:
This is equivalent to conditional rendering  `{featuredEvent  && <FeaturedPostCard {...featuredEvent} />}`